BaseFilter/sections/antiadblock.txt

```diff
@@ -12491,3 +12491,6 @@ dirtstyle.tv##.lightbox
 bigbrothercanada.ca#%#//scriptlet('set-constant', 'video_player_config.players.0.adblock', 'false')
 !+ PLATFORM(ios, ext_android_cb)
 @@||bigbrothercanada.ca^$generichide
+! https://github.com/AdguardTeam/AdguardFilters/issues/195226
+www.wikiwand.com##div[class^="wrapper_wrapper__"]
+www.wikiwand.com##div[class^="footer_wrapper__"]

```

Comment:
Will break the site:

<details>
<summary>Details</summary>

![image](https://github.com/user-attachments/assets/78f157d8-fba9-4370-b97c-e994808a152a)

</details>